mobile/apps/artist/templates/header.jade

```diff
@@ -1,11 +1,11 @@
--var hometown = artist.hometown || artist.get('hometown')
--var years = artist.years || artist.get('years')
--var name = artist.name || artist.get('name')
+-var hometown = artist.hometown || (typeof artist.get === 'function' ? artist.get('hometown') : null)
```

Comment:
Ah, so where these are Metaphysics-backed objects, but the data is blank, we would get the crash? Whoops.... :(